From 3f4683eb76bbf666def534029c0646e1ea4a4163 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Lureau?= <marcandre.lureau@gmail.com>
Date: Tue, 17 Apr 2012 14:32:38 +0200
Subject: [PATCH] hw/ac97: the volume mask is not only 0x1f

It's a case by case (see Table 66. AC ?97 Baseline Audio Register Map)

Signed-off-by: Marc-Andr? Lureau <marcandre.lureau@redhat.com>
Signed-off-by: malc <av1474@comtv.ru>
---
 hw/ac97.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/hw/ac97.c b/hw/ac97.c
index cd893c3..aa1babf 100644
--- a/hw/ac97.c
+++ b/hw/ac97.c
@@ -115,7 +115,6 @@ enum {
 #define EACS_VRA 1
 #define EACS_VRM 8
 
-#define VOL_MASK 0x1f
 #define MUTE_SHIFT 15
 
 #define REC_MASK 7
